osmaxx-py/config/settings/common.py

```diff
@@ -98,13 +98,19 @@
 EMAIL_HOST_PASSWORD = env.str('DJANGO_EMAIL_HOST_PASSWORD', default='')
 EMAIL_HOST_USER = env.str('DJANGO_EMAIL_HOST_USER', default='')
 EMAIL_PORT = env.int('DJANGO_EMAIL_PORT', default=25)
-EMAIL_SUBJECT_PREFIX = env.str('DJANGO_EMAIL_SUBJECT_PREFIX', default='')
+EMAIL_SUBJECT_PREFIX = env.str('DJANGO_EMAIL_SUBJECT_PREFIX', default='[Django] ')
```

Comment:
Don't know if we really should, it's only used for sending mails to admins and managers (special variants of Djangos `send_mail`). And since we're not using this anywhere, we can just use Django's default which is '[Django] '. But changing is OK too, I don't really care.